### ISSUE TITLE
Feature request: Faster animation/changing of the numbers 

### ISSUE THREAD

**Issue by luptilu** [kiln/template-line-chart-race#152]
A free user just remarked that they wished the numbers in the line chart race would change as quickly as in the bar chart race: 

https://secure.helpscout.net/conversation/950091979/7599/

Nothing urgent, but thought I would raise it.

As an example:

Here's a bar chart race with the quick animation of the numbers: https://app.flourish.studio/visualisation/657770/edit

And a line chart race with the same data: https://app.flourish.studio/visualisation/657765/edit

The numbers only seem to change every second.

Maybe this is something that could be added as a setting.

**Comment by theduncanclark** [comment]
Yes, definitely! I've wanted to do this for ages. I had an issue open for it already (#131) but with less detail so I've closed that one.
